pyproject.toml

```diff
@@ -27,40 +34,146 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Physics",
     "Topic :: Software Development :: Libraries :: Python Modules"
 ]
+dependencies = [
+    "pydantic~=2.5.2",
+    "h5py",
+    "numpy~=1.26.0",
+    "Pillow~=10.3.0",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+test = [
+    "coverage",
+    "pytest",
+    "pytest-cov",
+]
+quality = [
+    "pylint",
+    "pre-commit",
+]
 
-exclude = ["geoh5py/handlers/*", "geoh5py/interfaces/*"]
+[project.urls]
+Documentation = "https://mirageoscience-geoh5py.readthedocs-hosted.com/"
+Homepage = "https://www.mirageoscience.com/mining-industry-software/python-integration/"
+Repository = "https://github.com/MiraGeoscience/geoh5py"
 
+[tool.hatch.version]
+path = "geoh5py/__init__.py"
```

Comment:
**suggestion:** configure for auto-versioning from tags

This config here assign version from parsing `__version__` in __init__.py, right? Better than before, for sure!
Just one step further ;) can also be tackled in another issue.
Note that version is likely used again in doc. Is there a way to get it automated in the doc as well?

Reply:
We could do it with the package [`versiongit`](https://github.com/jwodder/versioningit). It is supported by hatch. Good idea, thank you!

Reply:
also https://github.com/ofek/hatch-vcs
See https://github.com/ofek/hatch-showcase